src/components/RecentForm/RecentForm.jsx

```diff
@@ -0,0 +1,18 @@
+import React from "react";
+import "./RecentForm.scss";
+
+const RecentForm = (props) => {
+  return (
+    <span className={`form ${props.data}`}>
+      {props.data.split("").map((game, i) => {
+        return (
+          <span key={`team.team_name__${i}`} className={`last_game ${game}`}>
```

Comment:
`key`s должны быть уникальны в пределах одного рендера. слишком сложное название кея просто не нужно.
можно сделать проще
```suggestion
          <span key={${game}-${i}`} className={`last_game ${game}`}>
```
и то сомневаюсь в целесообразности game добавлять.